package.json

```diff
@@ -313,7 +313,7 @@
     "grunt-npm": "0.0.2",
     "jasmine-core": "^2.3.4",
     "json3": "^3.3.2",
-    "karma-browserify": "^5.0.1",
+    "karma-browserify": "^5.0.2",
```

Comment:
Please fix this, as the travis is failing because this version is not released yet